src/main/java/com/recycle/buddy/controller/RecycleBuddyController.java

```diff
@@ -0,0 +1,26 @@
+package com.recycle.buddy.controller;
+
+import com.recycle.buddy.model.input.Image;
+import com.recycle.buddy.service.ImageRecognitionService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/images/v1")
+public class RecycleBuddyController {
+
+    private final ImageRecognitionService imageRecognitionService;
+
+    @Autowired
+    public RecycleBuddyController(final ImageRecognitionService imageRecognitionService) {
+        this.imageRecognitionService = imageRecognitionService;
+    }
+
+    @RequestMapping(value = "/recognize", method = RequestMethod.POST)
+    public String recognize(@RequestBody final Image image) {
```

Comment:
Didn't realize you were waiting on the front end for the API on this. I haven't talked with anyone else but I would argue for returning json in the form of something like
```
{ 
  "data":  { 
     "answers": ["Tin Can", "Aluminum Can" ],
     "minimumConfidence": 0.7 
  },
  "error": "error message"
 }
```
Only include the error field if there is an error. The answers can array can be 0, 1, or multiple entries.
You can do this in a future PR. Let's get this one in

Reply:
I was thinking about the similar structure .